.github/workflows/test.yaml

```diff
@@ -0,0 +1,26 @@
+name: Test Website Build
```

Comment:
Looks like this workflow is failing. 
Is the purpose to give a preview of the changes before merge? If the dev can do it locally, maybe we don't need this workflow?

Reply:
Yes, but the workflow is to ensure that any changes in the PR are not breaking the website build in the CI.

The test build is now working.